features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularContext.cpp

```diff
@@ -192,14 +191,29 @@ bool UBLOX_AT_CellularContext::activate_profile(const char *apn,
                 if (_at.get_last_error() == NSAPI_ERROR_OK) {
                     // Activate, wait upto 30 seconds for the connection to be made
                     _at.set_at_timeout(30000);
-                    _at.cmd_start("AT+UPSDA=0,3");
+                    _at.cmd_start("AT+UPSD=" PROFILE ",6,");
```

Comment:
Could you check `UPSDA` is dismissed here intentionally?

Reply:
This was a mistake! Thanks for pointing out, corrected.